provider/azure/instance.go

```diff
@@ -374,7 +375,7 @@ func (inst *azureInstance) IngressRules(ctx context.ProviderCallContext, machine
 
 		var portRange corenetwork.PortRange
 		if *rule.DestinationPortRange == "*" {
-			portRange.FromPort = 0
+			portRange.FromPort = 1
```

Comment:
Changed this to 1 as specifying a 0 port here triggers a port range validation error